src/components/Searchbar/index.jsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
+import './Searchbar.css';
+// import API from '../../services/API';
+import { getCity } from '../../redux/searchbar/searchbarSlice';
+
+const Searchbar = () => {
```

Comment:
- [ ] When searching for a city I can see the results in the redux logger, however, the view in the app doesn't change to reflect the new data, kindly make sure when searching for a city the data for that city is displayed in the app

Reply:
I just cloned the project, installed dependencies and tried it. Everything is working perfectly. Maybe you mistyped city name, if that's confusing for you, try biggest cities like Paris, London, Berlin etc...